### PR TITLE
Adds Internal Bleeding First-Aid Kits to Paramedic Stations

### PR DESCRIPTION
**Pull request by Shadowmech88**
The paramedic EVA gear locker has been renamed to paramedic gear.
The paramedic gear lockers on Box, Deff, and Meta each get one internal bleeding first-aid kit, containing four biofoam injectors.

This PR also corrects the color of the cross on the in-hand state of the advanced first-aid kit to be green instead of red.

:cl:
 * rscadd: The paramedic gear lockers on Box, Deff, and Meta each now have one internal bleeding first-aid kit, which contains four biofoam injectors.
 * rscadd: Paramedics now have a biofoam injector placed in their pocket when they spawn.

### CODE REVIEW

**Comment by Probe1** [comment]
😕 

I feel like that's super strong and it will get raided by the crew.  Can't we put it on paramedics when they join and drop the number a bit too?

**Comment by jakmak** [comment]
Probe you overestimate the crews competence. After the first few days people will forget it exists and paramedics will be the same old doctors with gimped access and only the best of the best medbay players would even acknowledge it's existence. Still it's a cool feature

**Comment by aceedex** [comment]
This is real dope, but I also agree with probe to an extent. Also, I think it should be a little more secure. Unique items like that usually cause break ins, so I'd eager put them in the EVA locker. 

Maybe 1 injector on any joining Para and 3-4 in the kit

**Comment by Fruktlimpa** [comment]
Make them not scannable by the device analyzer and make them unique in the sense that you cannot order more, because I would without a doubt pocket two of them at all times as the CEEMO if I knew they existed and weren't limitless.

**Comment by aceedex** [comment]
I'd disagree on the scanning them part. I feel like the best niche a mechanic could fill is to reproduce items with no other means to do so. 

But I don't see a problem with sticking MECH_SCAN_FAIL on it until you can set a specific cost for it

**Comment by Anton-Kr** [comment]
Yeah at worst you can fix popped lungs in like 6 minutes if you are slow and without an incision manager. Id argue against giving something to the paramedics and not adding the kit to medbay storage tbh

**Comment by aceedex** [comment]
On the other hand, anything that helps differentiate paramedic from MD but with shitty access is tip top stuf for me. Specially something like this that helps a lot in the first responder role the paramedic is supposed to be

**Comment by Anton-Kr** [comment]
Yeah but it doesnt make so much sense to give an advanced medication to a typically newer player, and someonw with less responsibilities and knowledge fluff wise. Parameds realistically at best need more actual rescue gear versus meds that straight up gimp the only surgical emergency the MDs deal with.

**Comment by Shadowmech88** [comment]
The idea behind paramedics getting these is so they can prevent patients from internally bleeding out as they're transported to Medbay, where the bleeding can actually be fixed.

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 1]
```diff
@@ -113,6 +113,27 @@
...
+	if (empty)
+		return
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
+	return
```
Comment (ghost) [inline code/game/objects/items/weapons/storage/firstaid.dm]: Remove

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 2]
```diff
@@ -113,6 +113,27 @@
...
+	desc = "Used to stabilize patients suffering from internal bleeding."
+	icon_state = "internalbleedfirstaid"
+	item_state = "firstaid-internalbleed"
+
+
+/obj/item/weapon/storage/firstaid/internalbleed/New()
+	..()
+	if (empty)
+		return
+	new /obj/item/weapon/reagent_containers/hypospray/autoinjector/biofoam_injector(src)
```
Comment (ghost) [inline code/game/objects/items/weapons/storage/firstaid.dm]: For loop this

File: code/game/objects/items/weapons/storage/firstaid.dm [hunk 3]
```diff
@@ -79,7 +79,7 @@
 	return
 
 /obj/item/weapon/storage/firstaid/o2
-	name = "oxygen deprivation first aid"
+	name = "oxygen deprivation first-aid kit"
```
Comment (ghost) [inline code/game/objects/items/weapons/storage/firstaid.dm:82]: But that's not atomic!

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Personally I think it's a little bit too powerful to give to parameds round start especially in this quantity

**Comment by 9600bauds** [comment]
It just temporarily stops bleeding.

That sounds a lot better than it actually is. The guy probably already bled out by the time you get to him, and he still needs medical attention anyways.

Heck, you can already effectively stop internal bleeding via 40ubicard10uAT pills.

**Comment by 9600bauds** [comment]
I agree with the limitation of 1 for each para + only 3/4 in the kit thou.

**Comment by jakmak** [comment]
Celt the only way to stay alive with internal bleeding long enough to get the surgery is a 50 pill of bicard+anti tox to deal with the OD, several hundred U of iron or copious amounts of dex/dex+ or any combination of the 3. Much like the previously mentioned methods only top tier medbay memers like yred remember those methods exist and those same people would be the only ones to use the paramedic only biofoam

**Comment by 9600bauds** [comment]
What makes you say paramedics wouldn't use their paramedic foam?

**Comment by 9600bauds** [comment]
About the locker's secure-ness: Lockers are pretty tough to greytide into, you need like 6 IEDs in average to crack them open. I'm only really concerned about CMOs looting.

**Comment by Shadowmech88** [comment]
The only reason I didn't put it in the secure locker in the first place was that it was specifically labeled "Paramedic EVA Gear," which wouldn't give the impression that there is medicine inside it.

**Comment by 9600bauds** [comment]
Oh, I thought it was in the locker

Put it in the locker or people will greytide for them

**Comment by Shadowmech88** [comment]
Should I rename the locker?

**Comment by 9600bauds** [comment]
I don't think anyone will mind

**Comment by Shadowmech88** [comment]
You mean if I do, or if I don't?

**Comment by 9600bauds** [comment]
If you do

If you add these to the lockers change the locker type instead of modifying the maps though

**Comment by Shadowmech88** [comment]
* The internal bleeding first-aid kit now contains just four biofoam injectors.
* The paramedic EVA gear locker has been renamed to paramedic gear.
* The internal-bleeding first-aid kit now spawns in the paramedic gear locker rather than out in the open.
* Paramedics now have a biofoam injector placed in their pocket when they spawn.

**Review by 9600bauds — APPROVED** [review]

**Comment by Fruktlimpa** [comment]
If it only temporarily stops internal bleeding and surgery is still required then I don't mind. I assumed that they fixed internal altogether.

**Comment by Intigracy** [comment]
Confrictu.

I still think it should get a recipe to create, probably something requiring multiple departments like medbots. (maybe make mint an ingredient in it :^)) Though that'd be a different PR.

**Comment by Shadowmech88** [comment]
Conflicts fixed.

**Comment by Probe1** [comment]
We can evaluate and rebalance if needed.

**Pull request merged**